### ISSUE TITLE
'How can we help you?' on homepage 

### ISSUE THREAD

**Issue by sophie-oms** [sophie-oms/Accessibility-audit#1]
Element has insufficient color contrast of 2.74 (foreground color: #00abc7, background color: #ffffff, font size: 27.0pt (36px), font weight: bold). Expected contrast ratio of 3:1
<img width="308" alt="Homepage issue 1" src="https://user-images.githubusercontent.com/77342915/104339269-51218400-54ef-11eb-88cf-121aa6204810.PNG">
